### PR TITLE
Change gossip frequency for node-related info

### PR DESCRIPTION
**Pull request by spencerkimball**
Reduce frequency to once-per-hour for node descriptor.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4227)

### CODE REVIEW

**Comment by tamird** [comment]
LGTM

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[server/node.go, line 54 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4227#-KA1PxfgtofAQ5KY98Lh:-KA1PxfgtofAQ5KY98Li:1353584056) ([raw file](https://github.com/cockroachdb/cockroach/blob/db3012212886285f504a0bd05788445af60c6600/server/node.go#L54)):*
call this `gossipNodeDescriptorInterval`. no need to be overly general when that's the only thing it applies to.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4227#-:-KA1QE2sM10YQyTSzZzc:676834233)*

File: server/node.go [hunk 1]
```diff
@@ -237,6 +239,7 @@ func (n *Node) initNodeID(id roachpb.NodeID) {
 	}
 	// Gossip the node descriptor to make this node addressable by node ID.
 	n.Descriptor.NodeID = id
+	log.Infof("setting node descriptor %+v", n.Descriptor)
```
Comment (bdarnell) [inline server/node.go]: Remove this?

Comment (spencerkimball) [inline server/node.go]: Yeah, probably makes sense.

Comment (tschottdorf) [inline server/node.go]: Looks like it was merged anyway.

**Pull request merged**